.github/contributing.md

```diff
@@ -96,6 +96,20 @@ Eslint config should not be overridden in any packages.
 
 Prettier config should not be overridden in any packages.
 
+#### GraphQL type generation
+
+[GraphQL Code Generator](https://the-guild.dev/graphql/codegen) is used for generating TypeScript types from `schema.graphql` files.
+
+```shell
+./<package>/.graphqlconfig # config
```

Comment:
I think the `auth` package does not have that...

Also, should we spell it out that this is where the config for GraphQL Code Generator is? I'm not 100% sure it's obvious.

Reply:
I'm actually not sure what `.graphqlconfig` is for and think we should just mention `codegen.yml`
https://github.com/interledger/rafiki/blob/main/packages/backend/codegen.yml